content/en/news/releases/1.10.x/announcing-1.10/change-notes/index.md

```diff
@@ -0,0 +1,174 @@
+- **Deprecated** installation flags `values.global.arch` in favor of the affinity Kubernetes settings.
+  ([Issue #30027](https://github.com/istio/istio/issues/30027))
+
+## Traffic Management
+
+- **Added** the `ISTIO_GATEWAY_STRIP_HOST_PORT` environment variable to control whether gateways strip host port before any processing of request by HTTP filters or routing. This option is disabled by default.
+  ([Blog](/blog/2021/discovery-selectors/), [Issue #25350](https://github.com/istio/istio/issues/25350))
+
+- **Added** `meshConfig.discoverySelectors` to dynamically restrict the set of namespaces for `Services`, `Pods`, and `Endpoints` that istiod processes when pushing xDS updates to improve performance on the data plane.
+  ([Issue #26679](https://github.com/istio/istio/issues/26679))
+
+- **Fixed** configuration of TLS parameters (TLS version, TLS cipher suites, curves, etc.) with `EnvoyFilter`.
+  ([Issue #28996](https://github.com/istio/istio/issues/28996))
+
+- **Fixed** an issue where filter chain name was ignored when processing `EnvoyFilter` match.
+  ([Issue #31166](https://github.com/istio/istio/issues/31166))
+
+- **Improved** the full push scoping by adding `Sidecar` config to `sidecarScopeKnownConfigTypes`.
+
+- **Improved** virtual machine integration to clean up `iptables` rules when the service is stopped.
+  ([Issue #29556](https://github.com/istio/istio/issues/29556))
+
+- **Updated** istio-proxy drain notification strategy to immediate from gradual.
+  ([Issue #31403](https://github.com/istio/istio/issues/31403))
+
+- **Added** CNI metrics counting repair operations.
+  ([Issue #19300](https://github.com/istio/istio/issues/19300))
+
+- **Added** `/debug/connections` istiod debug interface to list the current connected clients.
+  ([Issue #31075](https://github.com/istio/istio/issues/31075))
+
+- **Added** SDS secrets fetch failure metric `pilot_sds_certificate_errors_total`.
+  ([Issue #31779](https://github.com/istio/istio/issues/31779))
+
+- **Added** metrics for istiod informer errors.
+
+- **Added** 'istioctl validate' and the validating webhook now report duplicate or unreachable virtual service matches.
+  ([Issue #31525](https://github.com/istio/istio/issues/31525))
+
+- **Fixed** a bug where `ISTIO_META_IDLE_TIMEOUT` is not reflected when set to `0s`.
+  ([Issue #30067](https://github.com/istio/istio/issues/30067))
+
+- **Fixed** avoid unnecessary full push in service entry store.
+  ([Issue #30683](https://github.com/istio/istio/issues/30683))
+
+- **Fixed** a bug where the EnvoyFilter `HTTP_FILTER` didn't support `INSERT_FIRST`.
+  ([Issue #31573](https://github.com/istio/istio/issues/31573))
+
+- **Fixed** an issue where services with `PASSTHROUGH` load balancing were always sent mTLS traffic, even if the destinations did not support mTLS.
+  ([Issue #23494](https://github.com/istio/istio/issues/23494))
+
+- **Fixed** a bug where `EnvoyFilter` with service match did not work for inbound clusters.
+
+## Security
+
```

Comment:
`**Deprecated** the `values.global.jwtPolicy=first-party-jwt` option. This option is less secure and intended for backwards compatibility with older Kubernetes clusters without support for more secure token authentication, but  is now enabled by default in new Kubernetes versions. The option will be removed in future releases. See https://preliminary.istio.io/latest/docs/ops/best-practices/security/#configure-third-party-service-account-tokens for more info`

Reply:
Thanks, added